bucket/baidupcs-go.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "3.8.4",
+    "description": "Terminal utility for Baidu Network Disk",
+    "homepage": "https://github.com/qjfoidnh/BaiduPCS-Go",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/qjfoidnh/BaiduPCS-Go/releases/download/v3.8.4/BaiduPCS-Go-v3.8.4-windows-x64.zip",
+            "hash": "a9fdb978315b6e9c092f21d01393a7f826f8a9db9d7ea6d76e3af15c8d3d9277",
+            "extract_dir": "BaiduPCS-Go-v3.8.4-windows-x64"
+        },
+        "32bit": {
+            "url": "https://github.com/qjfoidnh/BaiduPCS-Go/releases/download/v3.8.4/BaiduPCS-Go-v3.8.4-windows-x86.zip",
+            "hash": "5d54a249d9f9d66e1caac8342b49a6474b2ac0c67bcd095848551a04fc612a20",
+            "extract_dir": "BaiduPCS-Go-v3.8.4-windows-x86"
+        }
+    },
+    "bin": "BaiduPCS-Go.exe",
+    "checkver": {
+        "github": "https://github.com/qjfoidnh/BaiduPCS-Go"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/qjfoidnh/BaiduPCS-Go/releases/download/v$version/BaiduPCS-Go-v$version-windows-x64.zip",
+                "extract_dir": "BaiduPCS-Go-v$version-windows-x64"
+            },
+            "32bit": {
+                "url": "https://github.com/qjfoidnh/BaiduPCS-Go/releases/download/v$version/BaiduPCS-Go-v$version-windows-x86.zip",
+                "extract_dir": "BaiduPCS-Go-v$version-windows-x86"
+            }
+        }
+    }
+}
```

Comment:
Any files to persist?

Reply:
For this package, configuration files stored in `%APPDATA%\BaiduPCS-Go`.

Reply:
Any way to move it to this directory?

Is the location hard coded for example?

Reply:
Not hard coded, it can be set by environment variable `BAIDUPCS_GO_CONFIG_DIR`.

Reply:
@rashil2000 should we try to:-
1. Set the BAIDUPCS_GO_CONFIG_DIR to a persisted directory within Scoop
2. Set persist for that specific directory in the manifest.

This would help maintain persistence

Reply:
It is possible. Rustup does this - https://github.com/ScoopInstaller/Main/blob/master/bucket/rustup.json

Reply:
@AlexaraWu use env_set if possible

https://github.com/ScoopInstaller/Main/blob/master/bucket/rustup.json#L25

You will have to decide on a directory that is not likely to be added by baidupcs-go.

Maybe something named persist

And then you will have to persist it like this

https://github.com/ScoopInstaller/Main/blob/master/bucket/rustup.json#L29

Reply:
Alright, I will add the persist block later.

Reply:
This package search for configuration file in install directory firstly, so just create an empty configuration file in install directory is fine.